core/services/editor/EditorBlockRegistrationManager.php

```diff
@@ -173,14 +175,16 @@ public function registerScripts()
     {
         wp_register_script(
             'ee-core-blocks',
-            $this->domain->distributionAssetsUrl() . 'ee-core-blocks.dist.js',
+            $this->assets_registry->getAssetUrl(Registry::ASSET_NAMESPACE, 'core-blocks', Registry::ASSET_TYPE_JS),
```

Comment:
this is great, but I still think we should add a couple of  helper methods to the assets Registry like:

 * getJsUrl( $namespace, $chunk);
 * getCssUrl( $namespace, $chunk);

which would both hit `getAssetUrl()` with the appropriate `ASSET_TYPE` constant

Reply:
That's an awesome suggestion!  Too bad you didn't make it in [this pull request](https://github.com/eventespresso/event-espresso-core/pull/287) where `getAssetUrl` was added :)

I think that's something we should do in a ticket off master (since its a helper that could get used elsewhere), thoughts?

Reply:
I did make that comment in that pull request, but then I added an additional comment afterwards and GH collapsed the original comment and marked it outdated. 

So then I added an additional comment to the entire thread stating:

>  I commented on some of my comments after thinking some more about things, so you'll need to click on the "Show Outdated" links to see everything I wrote (one set of comments is indeed outdated)

so you must have missed that somehow!

Reply:
Ya I missed the comment, definitely would have added it in the branch had I seen it. doh!